### PR TITLE
Updating Arch guidelines

### PR DESCRIPTION
**Pull request by Mazino-Urek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,14 @@ sudo apt-get install --install-recommends arc-kde
 
 or download .deb packages from [here](https://launchpad.net/~papirus/+archive/ubuntu/papirus/+packages?field.name_filter=arc-kde).
 
+### Arch and derivatives
+
+You can install adapta-kde from the official arch repo:
```
Comment (ArchangeGabriel) [inline README.md:39]: Copy paste error (adapta).

**Comment by ArchangeGabriel** [comment]
Same comment as in adapta BTW, official packages are the ones made by @varlesh AFAIU.

**Comment by Mazino-Urek** [comment]
I understand. You can close them.

**Comment by SmartFinn** [comment]
So, I'll rename "Unofficial packages" to "Third-party packages" to avoid the confusion. Also, I'll add the installation command for AUR packages.

**Pull request closed without merging**